cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -0,0 +1,1191 @@
+import json
+from datetime import datetime
+
+from django.test import TestCase
+from django.utils import timezone
+from django.utils.dateparse import parse_datetime
+
+from knowledgebase.management.commands._csv_2_fixture import KnowledgebaseCsvParse
+
+
+class TestCSV2Fixture(TestCase):
+    def setUp(self):
+        self.datetime_now = datetime.now().replace(tzinfo=timezone.get_current_timezone()).isoformat()
+
+    def compare_dicts(self, outputDict, expectedDict):
+        for key, value in expectedDict.items():
+            self.assertIsInstance(value, type(outputDict[key]))
+            if isinstance(value, int):
+                self.assertEqual(outputDict[key], value)
```

Comment:
This doesn't appear to differ from the fallback comparison (the one behind `else`). Is it needed for making this check go before the datetime one for some reason? Or an oversight?

Reply:
Yes it is needed for making it check before datetime. Parse_datetime from the django library only accepts string so when it sees an integer it fails.

Reply:
Oh I see. Hmm, that's a bit confusing. I think I'd prefer an explicit `elif isinstance(value, string) and isinstance(parse_datetime(value), datetime):` or something. I would never have guess the intent with it as it is

Reply:
Done

Reply:
Is it? I can't find it.

Reply:
Ohhh I just noticed the quote is one line, I mis-read. I will change that

Reply:
And also says `string` rather than `int`. It's a complete change to the logic of that section

Reply:
Now it's done